Lib/test/test_type_cache.py

```diff
@@ -0,0 +1,45 @@
+""" Tests for the internal type cache in CPython. """
+import unittest
+from collections import Counter
+from test import support
+try:
+    from sys import _clear_type_cache, _get_type_version_tag
+except ImportError:
+    _clear_type_cache = None
+
+msg = "requires sys._clear_type_cache and _get_type_version_tag"
+@support.cpython_only
+@unittest.skipIf(_clear_type_cache is None, msg)
```

Comment:
why not putting the string literal here? (msg)

Reply:
It would exceed 80 lines, and I didn't like how the multi-line decorator looked.

Reply:
You can write it on multiple lines:

```
skipIf(a,
       "long multi "
       "line string")

Reply:
Yeah I didn't like how that looked. Anyways, thanks to moving back to `_testcapi`, the message is shorter so I placed the message back.